R/databricks.R

```diff
@@ -0,0 +1,159 @@
+#' Helper for Connecting to Databricks via ODBC
+#'
+#' @description
+#'
+#' Connect to Databricks clusters and SQL warehouses via the [Databricks ODBC
+#' driver](https://www.databricks.com/spark/odbc-drivers-download).
+#'
+#' In particular, the custom `dbConnect()` method for the Databricks ODBC driver
+#' implements a subset of the [Databricks client unified authentication](https://docs.databricks.com/en/dev-tools/auth.html#databricks-client-unified-authentication)
+#' model, with support for personal access tokens, OAuth machine-to-machine
+#' credentials, and OAuth user-to-machine credentials supplied via Posit
+#' Workbench or the Databricks CLI on desktop.
+#'
+#' All of these credentials are detected automatically if present using
+#' [standard environment variables](https://docs.databricks.com/en/dev-tools/auth.html#environment-variables-and-fields-for-client-unified-authentication).
+#'
+#' @param http_path To query a cluster, use the HTTP Path value found under
```

Comment:
🤢 Should this be called `httpPath`? 🤢

Reply:
There are some snake-case arguments for functions in this package already (as well as camel-case and dot-separated ones :roll_eyes:), so it wouldn't be breaking precedent. But I'm happy to change it to camel-case if you wish.